src/components/main.tsx

```diff
@@ -1,23 +1,40 @@
 import React, { useState, useEffect } from "react";
 import BrightEthereumDeepLinkQR from "./qrGenerator";
-import { Button, Form, Input, Modal, ModalBody, Container, InputGroup, InputGroupAddon } from 'reactstrap';
-import { assignEthereum } from '../util/metamask'
+import { Button, Form, Input, Modal, ModalBody, Container, InputGroup, InputGroupAddon, FormFeedback } from 'reactstrap';
+import { assignEthereum, convertENS } from '../util/metamask'
 import MetaMask from '../assets/img/MetaMask.svg'
 import Web3 from 'web3';
 import { mobileCheck, androidOrIphoneLink } from "util/detectMobile";
 import { deepLinkPrefix } from "util/deepLink";
 
 const Main = () => {
     const [input, updateInput] = useState('');
+    const [resolvedENS, updateResolved] = useState('');
     const [showQR, toggleShowQR] = useState(false);
     const [notSupported, toggleNotSupported] = useState(false);
     const [validAddress, isValid] = useState(false);
     const [isMobile] = useState(mobileCheck())
     const [ethereum] = useState(() => assignEthereum());
     const [brightIdStoreLink] = useState(androidOrIphoneLink());
 
-    const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const handleChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
         updateInput(e.target.value)
+        const {value} = e.target
+        updateInput(value)
+
+        if( !value.includes('.eth') && Web3.utils.isAddress(value) ){
+          const address = await convertENS(value)
+          console.log('resolved ENS Address', address)
+          if(address && address.includes('.eth'))
+            updateResolved(address)
```

Comment:
Same as above, since `resolvedENS` is never used, `updateResolved` has no purpose